Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM clojure:lein-2.7.1-alpine
```

Comment:
What about setting up the YourKit agent to run by default? (It might be nice for data-processor down the road, and this would be an easy start down that path)